### PR TITLE
fix loss parallel parity test, spmd_types 0.2.1

### PR DESCRIPTION
**Pull request by pianpwk**
well this was stupid

(fwd was wrapped, interestingly GPU still passed)

### CODE REVIEW

**Review by wconstab — APPROVED** [review]
potentially unblock CI fail on main, confirm cpu integration test is clean first and then land

File: tests/unit_tests/test_loss.py [hunk 1]
```diff
@@ -391,11 +391,9 @@ def test_loss_parallel_cross_entropy_parity(self):
...
                         stride=(vocab_size, 1),
                     )
-                    self.assertTrue(
-                        torch.equal(
-                            local_grad_dtensor.full_tensor(),
-                            logits_dtensor.grad.full_tensor(),
-                        )
+                    torch.testing.assert_close(
+                        local_grad_dtensor.full_tensor(),
+                        logits_dtensor.grad.full_tensor(),
```
Comment (tianyu-l) [inline tests/unit_tests/test_loss.py]: this doesn't smell right

**Review by tianyu-l — CHANGES_REQUESTED** [review]

**Review by tianyu-l — APPROVED** [review]
